base/server/share/conf/pki.policy

```diff
@@ -32,6 +32,10 @@ grant codeBase "file:/usr/share/java/ecj.jar" {
         permission java.security.AllPermission;
 };
 
+grant codeBase "file:/usr/share/java/ecj/ecj.jar" {
```

Comment:
Since this is a protected path -- I think it is safe to add it and leave the old path as well -- it depends on the version of an external package, which probably differs from system to system.

Reply:
We don't really need to worry about Debian paths since they usually patch the source code anyway, but regardless, this is fine.